uiowa/uiowa
# Started rabbit hole work

Closes #4446 
# How to test


1. blt ds --site=sandbox.uiowa.edu
2. enable area of study split
3. drush @sandbox.local cim
4. drush @sandbox.local cr
5. add an area of study - add a primary source link. If check 'link directly to source' the rabbit hole settings should change - redirect radio should be selected. If uncheck it, the 'display page' rabbit hole radio should be selected

# Stories

- As a site visitor I am redirected when a more suitable destination is configured.
- As an editor/webmaster I can set a primary source link (internal or external) and set link directly to source to have visitors redirected from the view page or by navigating directly to the node.
- As an editor/webmaster I can still edit the area of study node and not be redirected. I also see a message that visitors are being redirect.
